doc/sphinx/source/level0.rst

```diff
@@ -6,14 +6,15 @@ Level Zero driver
 
 This driver uses libze and LLVM/Clang to run OpenCL code on GPU devices via Level Zero API.
 
-The implementation is almost complete, there are a few remaining issues.
+The implementation is now (as of Mar 2025) able to pass OpenCL-CTS tests, however improvements
+to performance and functions are still ongoing.
 
 Installation
 -------------
 
 Required:
 
- * Clang+LLVM: 17, 18 and 19 should work, older may work but are untested.
+ * Clang+LLVM: 18, 19 and 20 should work, older may work but are untested.
```

Comment:
While we still actually test the older ones with CIs, let's list them?

Reply:
what do you mean ? those are the one we currently test with Level Zero CI: 18 to 20.

Reply:
Ah sorry, mislooked and though it was the list for PoCL-CPU.